bubblepicker/src/main/java/com/igalata/bubblepicker/physics/Engine.kt

```diff
@@ -66,7 +71,7 @@ object Engine : ContactListener {
             val y = Random().nextFloat() - 0.5f
             val vx = (if (Random().nextBoolean()) -0.01f else 0.01f) * Random().nextFloat() / scaleY
             val vy = (if (Random().nextBoolean()) -0.01f else 0.01f) * Random().nextFloat() / scaleY
-            bodies.add(CircleBody(world, Vec2(x, y), bubbleRadius * scaleX, (bubbleRadius * scaleX) * 1.1f, density, Vec2(vx, vy)))
```

Comment:
I suppose the initial idea was to make the size of bubble dependant on one of the dimension, but the `radius` is later modified again based on `scaleX` and `scaleY`, so the effect has been exponential. In our app the effect was following - the bigger the ratio of height to width was, exponentially bigger the bubbles were. 

I just removed the "exponential" part, so bubble size is affected by ratio, not ratio^2.